packages/plans-grid-next/src/types.ts

```diff
@@ -153,6 +155,7 @@ export interface GridAction {
 		callback: () => void;
 		// TODO: It's not clear if status is ever actually set to 'blocked'. Investigate and remove if not.
 		status?: 'disabled' | 'blocked' | 'enabled';
+		variant?: 'primary' | 'secondary';
```

Comment:
Nit: I'm wondering if we should follow the `Button` component's convention of keeping it as a "primary" boolean prop: https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/button/README.md#L25

I'm fine with us staying with this though, since it fits better to the `status` prop design above; just a soft comment :)

Reply:
My rationale behind this decision is that the hook returns a data structure representing a `GridAction`. I believe that the responsibility of rendering the `GridAction` should lie with the display component. Users of this hook may not always want to use the button from `@automattic/components`; they might prefer a different button implementation. So, I decided to make this property generic to give display components the flexibility to render it as they see fit.